GVRf/Framework/framework/src/main/java/org/gearvrf/scene_objects/GVRGUISceneObject.java

```diff
@@ -0,0 +1,175 @@
+package org.gearvrf.scene_objects;
+
+import android.os.Handler;
+import android.os.Message;
+import android.view.InputDevice;
+import android.view.KeyEvent;
+import android.view.MotionEvent;
+import android.view.View;
+
+import org.gearvrf.GVRBaseSensor;
+import org.gearvrf.GVRContext;
+import org.gearvrf.GVRMesh;
+import org.gearvrf.GVRMeshCollider;
+import org.gearvrf.GVRPicker;
+import org.gearvrf.ISensorEvents;
+import org.gearvrf.SensorEvent;
+import org.gearvrf.scene_objects.view.GVRView;
+
+import java.util.List;
+
+
+/**
```

Comment:
It would be real nice if you could give a high-level overview what is this all about; what it does for users; short explanation how to use. Thanks.